exercises/I-floats/exercise.js

```diff
@@ -1,2 +1,8 @@
 var numberOfStudents = 15;
 var numberOfMentors = 8;
+
+let studentPercent = (numberOfStudents * 100)/(numberOfStudents + numberOfMentors);
+let mentorsPercent = (numberOfMentors * 100)/(numberOfStudents + numberOfMentors);
+
+console.log("Percentage students:", Math.round(studentPercent));
```

Comment:
Very well and readable, the only comment I can give is if you ad the % sign to the final result, it will be more like expected result